Lib/Protocols/IdFTP.pas

```diff
@@ -3983,22 +3978,20 @@ function TIdFTPClientIdentifier.GetClntOutput: String;
         Result := Result + ' ' + FPlatformDescription;
       end;
     end;
-  end else begin
-    Result := '';
   end;
 end;
 
 function TIdFTPClientIdentifier.GetCSIDOutput: String;
 begin
-  Result := '';
+  Result := 'Name='+gsIdProductName + '; Version=' + gsIdVersion;

```

Comment:
Why put the default here instead of in an `else` block when `FClientName` is empty?